kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/rules/temp/drugbank/drugbank_rules.clj

```diff
@@ -1,99 +1,101 @@
-   ;; (_/fv1 obo/IAO_0000219 _/externalid)
-   
-   (_/fv1 kiao/hasTemplate iaodrugbank/DrugBankPartnerRecord_externalIdentifiersDataField1)
-   (_/fv1 obo/IAO_0000219 _/externalid)
-   (_/partnerRecord1 obo/BFO_0000051 _/fv1) ; has_part
-   (_/targetRecord1 obo/BFO_0000051 _/partnerRecord1) ; has_part
-   (_/record obo/BFO_0000051 _/targetRecord1) ; has_part
-
-   (_/record obo/BFO_0000051 _/fv0) ; has_part
-   (_/fv0 kiao/hasTemplate iaodrugbank/DrugBankDrugRecord_drugBankIdDataField1)
-   (_/fv0 obo/IAO_0000219 _/drugid) ; denotes
+    ;; each DrugBank record has an ID field. We use that
+    ;; ID to link to the drug bioentity
+    (_/id_field kiao/hasTemplate iaodrugbank/DrugBankDrugRecord_drugBankIdDataField1)
+    (_/id_field obo/IAO_0000219 _/drug_id)
+  	(_/drug_id obo/IAO_0000219 ?/drug_bioentity)
 
-   (_/drugid obo/IAO_0000219 ?/drug) ; denotes
+    ;; the majority of DrugBank records link to a Target record.
+  	(_/base_record obo/BFO_0000051 _/id_field)
+  	(_/base_record obo/BFO_0000051 _/target_record)
+  	(_/target_record kiao/hasTemplate iaodrugbank/TargetSchema1)
 
-   (_/externalid obo/IAO_0000219 ?/bio) ; denotes
-   (?/bio [rdfs/subClassOf clojure.core/*] ?/gorgporv)
-   (?/gorgporv rdf/type kbio/GeneSpecificGorGPorVClass))
+    ;; the vast majority of targets contain a polypeptide record which
+    ;; links to a protein bio-entity
+    ;; of 12,704 target records, all but 225 have an associated polypeptide record
+    ;; we should look into those 225 targets at some point as they are currently excluded
+    ;; There is card for this: https://trello.com/c/JulbRanl
+    (_/target_record obo/BFO_0000051 _/polypeptide_record)
+    (_/polypeptide_record kiao/hasTemplate iaodrugbank/PolypeptideSchema1)
+    (_/polypeptide_record obo/BFO_0000051 _/polypeptide_id_field)
+    (_/polypeptide_id_field obo/IAO_0000219 _/protein_ice_id)
+    (_/protein_ice_id obo/IAO_0000219 ?/target_bioentity)
+  )
 
-  
-  :reify ([?/interaction {:ln (:sha-1 "interaction" ?/bio ?/drug)
+  :reify ([?/interaction {:ln (:sha-1 "interaction" ?/target_bioentity ?/drug_bioentity)
                           :ns "kbio" :prefix "I_"}]
-          [?/r1 {:ln (:restriction)
-                 :ns "kbio" :prefix "R_"}]
-          [?/r2 {:ln (:restriction)
-                 :ns "kbio" :prefix "R_"}]
-          [?/r3 {:ln (:restriction)
-                 :ns "kbio" :prefix "R_"}]
-          [?/r4 {:ln (:restriction)
-                 :ns "kbio" :prefix "R_"}]
-          [?/inheres {:ln (:restriction)
-                      :ns "kbio" :prefix "R_"}])
+          [?/target {:ln (:sha-1 ?/target_bioentity ?/drug_bioentity)
+                    :ns "kbio" :prefix "T_"}]
+          [?/drug {:ln (:sha-1 ?/target_bioentity ?/drug_bioentity)
+                    :ns "kbio" :prefix "D_"}]
+          [?/r_target_participant {:ln (:restriction)
+                 :ns "kbio" :prefix "RESTR_"}]
```

Comment:
Actually, I suppose this doesn't really affect querying too much because we match restrictions by type, rather than by name.  The inconsistency would come into play more when a human is exploring using Gruff or something similar.  
I'm okay with this going in, but we should add a card to bring other restrictions in line with the new convention.